src/wazuh_db/helpers/wdb_global_helpers.c

```diff
@@ -955,40 +941,6 @@ int wdb_delete_agent_belongs(int id, int *sock) {
     return result;
 }
 
-int wdb_remove_group_from_belongs_db(const char *name, int *sock) {
```

Comment:
If we are removing `wdb_remove_group_from_belongs_db()`, it would be cleaner to also remove all its UT.

Reply:
They are removed now.